common/src/main/java/tachyon/worker/WorkerNetAddress.java

```diff
@@ -40,6 +41,18 @@ public WorkerNetAddress(String host, int rpcPort, int dataPort, int webPort) {
   }
 
   /**
+   * Constructs the worker net address from thrift construct.
+   *
+   * @param netAddress the thrift net address
+   */
+  public WorkerNetAddress(NetAddress netAddress) {
```

Comment:
does this need to be called WorkerNetAddress if it is under tachyon.worker namespace? (it wraps the tachyon.thrift.NetAddress so we could just use that one fully qualified here and the "worker" NetAddress everywhere else)